.gitignore

```diff
@@ -1,2 +1 @@
 /node_modules
-/dist
```

Comment:
We need to keep `/dist` on `.gitignore`

Reply:
Included in e8e98e2065fc0310fbe1fb02ac454c8199f5f7f2